README.md

```diff
@@ -0,0 +1,158 @@
+<div align="center">
+
+  <h3><b>Vet_clinic_Schema_Diagram
+</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
```

Comment:
- [ ] Hello, Kindly make sure to remove the `Live Demo` link from the headers table of content.